src/main/java/com/telefonica/iot/perseo/Configuration.java

```diff
@@ -56,24 +62,62 @@ private Configuration() {
      * @return true if success, false otherwise
      */
     public static synchronized boolean reload() {
+
         LOGGER.info("Configuration is being reloaded");
         InputStream stream;
+        Long default_max_age;
```

Comment:
As far as I understand, Perseo Core code style uses camelCase for variable names.

Reply:
Ok. Do you have a guide with perseo style rules that I can study before continuing with other PR? I see telefonica_checkstyle.xml file. What tool is meant to be used for checking the perseo-core style using this file?

Reply:
I think telefonica_checkstyle.xml is used as input by [Maven Checkstyle plugin](https://maven.apache.org/plugins/maven-checkstyle-plugin).

(Not fully sure... 90% sure :)